exec/java-exec/src/test/java/org/apache/drill/exec/vector/complex/writer/TestExtendedTypes.java

```diff
@@ -26,11 +26,11 @@
 import org.apache.drill.test.BaseTestQuery;
 import org.apache.drill.exec.ExecConstants;
 import org.apache.drill.exec.rpc.user.QueryDataBatch;
-import org.junit.Assert;
 import org.junit.BeforeClass;
 import org.junit.Test;
 
 public class TestExtendedTypes extends BaseTestQuery {
+
```

Comment:
There is some cleanup in `TestComplexTypeWriter.java` to be copied across.

Reply:
Spaces? Could you point me what cleanup do you mean?

Reply:
See [this diff](https://github.com/paul-rogers/drill/compare/DRILL-6953-rev2...vdiravka:DRILL-8037#diff-343fdacd56be76f487ce9e5146b186dd3cfbd6fa3dce74c420941242605630f7), then search for `TestComplexTypeWriter`. You'll see some minor differences.